os/board/common/README.md

```diff
@@ -0,0 +1,114 @@
+# How to partition a flash into a config
+
+This document provides a way to register a partial block driver in Flash.
+
+## Prepare
+
+To partition Flash, you must first enable ARCH_USE_FLASH. So if new board use FLASH, add 'select ARCH_USE_FLASH' to Kconfig. 
+
+## Contents
+
+> [How to Configure CONFIG](#how-to-configure-config)  
+> [Partition size](#Partition-size)  
+> [Partition type](#partition-type)  
+> [Partition name](#partition-name)  
+
+## How to Configure CONFIG
+
+All partition information is separated by comma and the last must end with a comma. The information of FLASH_PART_SIZE, FLASH_PART_TYPE and FLASH_PART_NAME should be in order. 
+
+ex)
+
+FLASH_PART_SIZE "16,48,192,32,512,2400,1536,1536,1400,8,512,"  
+FLASH_PART_TYPE "none,none,none,none,none,none,none,none,smartfs,config,none,"  
+FLASH_PART_NAME "bl1,sssro,bl2,sssfw,wlanfw,os,factory,ota,user,nvram,sssrw,"  
+
+name    | type    | size
+------- | ------- | --------
+bl1     | none    | 16
+sssro   | none    | 48
+bl2     | none    | 192
+sssfw   | none    | 512
+wlanfw  | none    | 2400
+os      | none    | 1536
+factory | none    | 1536
+user    | smartfs | 1400
+nvram   | config  | 8
+sssrw   | none    | 512
+
+
+## Partition size
+
+FLASH_PART_SIZE is Flash partition size. The unit of partition size is KB.  
+If you want to change partition size, you can change it at below location.  
```

Comment:
You can find and modify it using menuconfig as shown below:
'''
Board ...
'''

Reply:
Okay. I will fix.